wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/ConversationGuestOptionsViewModel.swift

```diff
@@ -306,7 +332,7 @@ final class ConversationGuestOptionsViewModel {
                 switch result {
                 case .success:
                     self.updateRows()
-                    if self.link == nil && allowGuests {
+                    if (self.link == nil || self.securedLink == nil) && allowGuests {
```

Comment:
Not sure, but shouldn't be (self.link == nil **&&** self.securedLink == nil) ?
For example, `self.securedLink` is NOT nil and `link` will be nil. I believe we don't need to fetch it again is we use secure link.